### DISCUSSION TITLE
Argument name= not working in class Text?

### DISCUSSION THREAD

**Discussion by ubaldot** [ManimCommunity/manim#3170] [Q&A]
If I run

```
t = Text("foo", name="bar")
self.add(t)
print(self.mobjects)

```
I get `[Text('foo')]` whereas I would expect `[bar]`. 

Is it a bug or I am  misunderstanding something here?

**Comment by MrDiver** [comment]
The convention for python objects is that the printed representation of an Object should represent the Constructor call that it can be created with. So in that case it will be `Text("foo")` because that is the Constructor that needs to be called in order to create an object that contains the text "foo".

But it might be different for the classes because not all of them were written in the same timespan.
